[.bat

```diff
@@ -0,0 +1,16 @@
+@echo off
+
+if "%1"=="-f" goto exists
+goto end
+
+:exists
+if EXIST %2. (
+	echo exists %2
+	exit /b 0
+) ELSE (
+	echo doesn't exist %2
+	exit /b 1
+)
+goto end
+
+:end
```

Comment:
Likewise, I would prefer remove the automatic bench data downloading rather than complicate this file with cross-platform shell hacks.